src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -51,12 +53,84 @@ public sealed class NpgsqlRawCopyStream : Stream, ICancelable
 
         #region Constructor
 
-        internal NpgsqlRawCopyStream(NpgsqlConnector connector, string copyCommand)
+        private async Task SendQueryAsync(NpgsqlConnector connector, NpgsqlStatement statement, bool async)
+        {
+            await connector.ParseMessage
+                .Populate(statement.SQL, statement.StatementName, statement.InputParameters, connector.TypeMapper)
+                .Write(_writeBuf, async);
+            var bind = connector.BindMessage;
+            bind.Populate(statement.InputParameters, "", statement.StatementName);
+            await connector.BindMessage.Write(_writeBuf, async);
+            await connector.DescribeMessage
```

Comment:
I don't think it makes sense to send a Describe, since there's no result set coming back.